lisp/ui/widgets/dbmeter.py

```diff
@@ -77,6 +87,35 @@ def plot(self, peaks, _, decayPeak):
 
         self.update()
 
+    def updateMarkings(self):
+        self._markings = []
+
+        height = self.height()
+        font_height = QFontMetrics(self.font()).height()
+        steps = [1, 5, 10, 25, 50, 100, 250]
+        curr_level = self.dBMax
+        curr_y = 0
+
+        while curr_y < height - font_height:
+            prev_level = curr_level
+            prev_y = curr_y + font_height
+
+            for step in steps:
+                curr_level = prev_level - step
+                curr_y = height - self.scale(curr_level) * (height - 2)
+                if curr_y > prev_y:
```

Comment:
We should take in account that the text will be vertically centered
```suggestion
                if curr_y - ceil(font_height / 2) > prev_y:
```

Reply:
Done. Differently, but done.